src/sentry/conf/server.py

```diff
@@ -1844,6 +1844,7 @@ def build_cdc_postgres_init_db_volume(settings):
                 "REDIS_HOST": "{containers[redis][name]}",
                 "REDIS_PORT": "6379",
                 "REDIS_DB": "1",
+                "ENABLE_SENTRY_METRICS_DEV": "1" if settings.SENTRY_USE_METRICS_DEV else "",
```

Comment:
It's either this, or run the metrics consumer in snuba devserver unconditionally, i.e. remove the following condition:
https://github.com/getsentry/snuba/blob/beb543bd505994daf3dff2f1f5632ecd53515991/snuba/cli/devserver.py#L140
@MeredithAnya what do you think?

Reply:
yeah this looks good to me!